### PR TITLE
Autobuild with GitHub Actions

### PR DESCRIPTION
**Pull request by Siddhant-Ray**
TL;DR : CI for auto-building changes to resume on every push to main.

Hi,

I happened to see you're hosting your resume .tex files and keeping an updated pdf on GitHub. I would suggest using this GitHub actions CI for autobuild.

Why?

1. Every push builds a new release, which creates a new PDF binary. This release is permanently stored and you don't need to rely on linking a built PDF from an external site.
2. It will ensure that every change is hosted as a new PDF, and you don't have to manually track the version from the website.

After merging, you must update the release path for the compiled PDF binary in your README.

### CODE REVIEW

**Comment by sloorush** [comment]
Hey @Siddhant-Ray! Thank you so much for your contribution!

So my basic workflow with the tex file was to edit and update it on overleaf and then export it and push it.

I guess this PR should make things simpler! Thanks 😃

**Comment by sloorush** [comment]
So I also maintain this repository where I store files to be hosted. I also used to push the pdf binary to that repo manually every time there was an update.

I guess we can automate that as well since the binary is anyways built on GitHub actions now.
